L1Trigger/L1TCommon/src/Parameter.cc

```diff
@@ -0,0 +1,115 @@
+#include "L1Trigger/L1TCommon/interface/Parameter.h"
+
+using namespace std;
+
+namespace l1t{
+
+Parameter::Parameter(const char *id,
+                     const char *procOrRole,
+                     const char *type,
+                     const char *value,
+                     const char *delimeter
+                    ){
+    this->id = id;
```

Comment:
Thanks for pointing to the naming convention. I'd stick with the 'this->' construct for the moment so as to minimize the changes.